File: app/src/main/java/com/jesusc24/xroadsthroughthecastle/ui/about/AboutUsFragment.java
package com.jesusc24.xroadsthroughthecastle.ui.about;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

import com.jesusc24.xroadsthroughthecastle.R;
import com.vansuita.materialabout.builder.AboutBuilder;
import com.vansuita.materialabout.views.AboutView;

/**
 * Fragment que muestra el AboutUs que se usa una libería de terceros para poder hacerlo
 */
public class AboutUsFragment extends Fragment {

    private static final String TAG = "SendMesssageProject";

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {

        AboutView view = AboutBuilder.with(getActivity())
                .setPhoto(R.mipmap.profile_picture)
                .setCover(R.mipmap.profile_cover)
                .setName("Jesús Cortés Ocaña")
                .setSubTitle(R.string.aboutUs_subTitle)
                .setBrief(R.string.aboutUs_brief)
                .setAppIcon(R.mipmap.img_icono)
                .setAppName(R.string.app_name)
                .addGitHubLink("https://github.com/JesusC24/")
                .addTwitterLink("https://twitter.com/jesuscortes2408")
                .addFiveStarsAction()
                .setVersionNameAsAppSubTitle()
                .addShareAction(R.string.app_name)
                .setWrapScrollView(true)
                .setLinksAnimated(true)
                .setShowAsCard(true)
                .build();
        return view;
    }
}

File: app/src/main/java/com/jesusc24/xroadsthroughthecastle/ui/about/AboutUsFragment.kt
package com.jesusc24.xroadsthroughthecastle.ui.about

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import com.jesusc24.xroadsthroughthecastle.R
import com.jesusc24.xroadsthroughthecastle.data.constantes.Constants
import com.jesusc24.xroadsthroughthecastle.utils.PreferencesManager
import com.vansuita.materialabout.builder.AboutBuilder

class AboutUsFragment : Fragment() {

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {

        return AboutBuilder.with(activity!!)
            .setPhoto(R.mipmap.profile_picture)
            .setCover(R.mipmap.profile_cover)
            .setName(R.string.my_name)
            .setSubTitle(R.string.aboutUs_subTitle)
            .setBrief(R.string.aboutUs_brief)
            .setAppIcon(R.mipmap.img_icono)
            .setAppName(R.string.app_name)
            .addGitHubLink("JesusC24")
            .addTwitterLink("jesuscortes2408")
            .addLinkedInLink("jesús-cortés-ocaña-359631221")
            .addFiveStarsAction()
            .setVersionNameAsAppSubTitle()
            .addShareAction(R.string.app_name)
            .setWrapScrollView(true)
            .setLinksAnimated(true)
            .setShowAsCard(true)
            .build()
    }

    override fun onResume() {
        super.onResume()
        val preferenceManager = PreferencesManager(context)
        preferenceManager.putBoolean(Constants.KEY_AVAILABILITY, true)

    }

    override fun onPause() {
        super.onPause()
        val preferenceManager = PreferencesManager(context)
        preferenceManager.putBoolean(Constants.KEY_AVAILABILITY, false)
    }
}